webhooks-extension/docs/GettingStarted.md

```diff
@@ -0,0 +1,161 @@
+- git credentials must be stored in a secret with an annotation, `tekton.dev/git-0: [url to git repository]`
+- docker registry credentials must be stored in a secret with an annotation, `tekton.dev/docker-0: [url to docker registry]`
+- credentials are located via the service account used by a given Tekton TaskRun or PipelineRun
+See the [Tekton documentation](https://github.com/tektoncd/pipeline/blob/master/docs/auth.md#basic-authentication-git) for more information. 
+
+Before you start creating Tekton resources, decide which Kubernetes namespace(s) you wish to execute pipelines in, and which service accounts will be used to execute them. Ensure that each service account has sufficient permissions under Kubernetes [Role-Based Access Control (RBAC)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) to execute your intended pipelines.
+
+Having set up the right namespaces and service accounts you can now proceed to create the necessary secrets. The Tekton dashboard has a 'Secrets' menu that can help with the creation of credentials for use within Tekton pipelines. 
+
+### Create credentials: Git
+
+You will need to create a secret for Git if any of your GitHub repositories require authentication in order to clone them. This is true for GitHub private repositories, and for many GitHub Enterprise installations. While Tekton supports SSH and Basic authentication for Git, only the latter is supported by the webhooks extension today. Basic authentication can be used to provide either username/password, or an access token. Access tokens are our recommended approach. 
+
+Create an access token via GitHub. As of August 2019, for a given GitHub repository, visit [your name icon in the top right corner] > Settings > Developer settings > Personal access tokens > Generate new token and create a token with `repo` permissions. Add `admin:repo_hook` permissions if you are going to use the same access token to create webhooks. 
+
+Store the access token in a Kubernetes secret via the Tekton Dashboard's 'Secrets' menu. This panel will help you add the right annotation to the secret, and patch it onto the specified service account. Annotations should typically be of the form, `tekton.dev/git-0: https://github.com`. If you are using an access token, store it in the 'password' field. Be sure to specify the service account that will be used by the associated Tekton pipeline. 
+
+![Creating a Git secret](createGitSecret.png?raw=true "Creating a Git secret")
+
+### Create credentials: Docker
+
+Credentials are normally required to push images to a docker registry. Some docker registries support access tokens. Others such as docker hub only support username/password. If you are pushing images to Docker Hub the annotation should be of the form, `tekton.dev/docker-0: https://index.docker.io/v1/`. Again, select the service account of the associated Tekton pipeline so that it is correctly patched with the new docker credentials. 
```

Comment:
Nit: inconsistent references to Docker Hub, suggest we stick with `Docker Hub` everywhere it's mentioned, and capitalise the D in `docker` mentions

Reply:
done